nautobot_circuit_maintenance/models.py

```diff
@@ -241,27 +241,33 @@ class NotificationSource(OrganizationalModel):
 
     # Mark field as private so that it doesn't get included in ChangeLogging records!
     _password = encrypt(models.CharField(max_length=100))
-    source_id = models.EmailField(max_length=100, unique=True, help_text="Email address used as identifier.")
-    url = models.CharField(max_length=200)
+    source_id = models.EmailField(
+        max_length=100, unique=True, help_text="Identifier (i.e. email address) to use to authenticate."
```

Comment:
Possibly add `verbose_name="Source ID"` so that the form presents it as such instead of as `Source id`?